notebook/static/base/js/i18nload.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Plugin to load a single locale.
+ */
+define([
+	"require",
+	"module",
+	 // These are only here so that the optimizer knows which ones we MIGHT load.
+	 // We will actually only load the ones we need. There should be one entry
+	 // here for each language you want to support.
+	 // For example, for German....
+	 // "json!base/../../i18n/de/LC_MESSAGES/nbjs.json"
+    ], function (require, module) {
+	return {
+		id: module.id,
+
+		load: function (locale, callerRequire, onload, loaderConfig) {

```

Comment:
The load function is used as it's what actually loads the right data ( i.e. it's the guts of the plugin ).  As discussed before, the plugin has two parts, the module ID ( stuff before the ! ) and the argument ( which in this case is "locale" the first argument passed to the load function.